### PR TITLE
Sync with Liguist v7.13.0

### PR DESCRIPTION
**Pull request by lafriks**
https://github.com/github/linguist/releases/tag/v7.13.0

### CODE REVIEW

File: common_test.go [hunk 1]
```diff
@@ -460,7 +458,7 @@ func (s *EnryTestSuite) TestGetLanguageGroup() {
 		expected string
 	}{
 		{name: "TestGetLanguageGroup_1", language: "BestLanguageEver", expected: ""},
-		{name: "TestGetLanguageGroup_2", language: "JSX", expected: "JavaScript"},
+		{name: "TestGetLanguageGroup_2", language: "Bison", expected: "Yacc"},
```
Comment (bzz) [inline common_test.go:461]: I'm a bit confused about why is this change needed? Did the mock data we use changed elsewhere?

**Review by bzz — COMMENTED** [review]
Thanks the patch! 
LGTM, sans a question on changing the test.

File: common_test.go (hunk 1 above)
Comment (lafriks) [reply common_test.go:461]: JSX was removed as language and `.jsx` extension was just added to `JavaScript`

Comment (bzz) [reply common_test.go:461]: Did not realise that - thank you so much for explaining!

**Comment by lafriks** [comment]
Would be nice to 🔪 new release now :)
